USSF-ORBIT/ussf-portal-client
# feat: Add Apollo Server and resolvers for My Space functionality

## Description 

Sets up Apollo Server, MongoDB connection, and resolvers.

Fixes #274 

## Review Notes
- [x] Fix Cypress tests
- [x] Fix Storybook build
- [x] Clean up types for consistency 

To test, run:

`docker compose up`

You can view the database GUI at localhost:8888 and check the `users` collection to confirm your changes in the portal are saving.

#288 should be merged first
